src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,110 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final int LOGIN_MIN_LENGHT = 6;
+    private static final int USER_PWD_MIN_LENGHT = 6;
+    private static final int USER_MIN_AGE = 18;
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("Alice66");
+        user.setAge(20);
+        user.setPassword("QWERT666");
+    }
+
+    @Test
+    void get_allData_Ok() {
+        user.setLogin("Alice66");
+        user.setAge(20);
+        user.setPassword("QWERT666");
+        assertEquals("Alice66", user.getLogin());
+        assertEquals(20, user.getAge());
+        assertEquals("QWERT666", user.getPassword());
+    }
+
+    @Test
+    void get_userNull_notOk() {
```

Comment:
```suggestion
    void register_nullUser_notOk() {
```

incorrect method name

would be more correct